py/pysparkling/context.py

```diff
@@ -103,6 +104,30 @@ def getOrCreate(spark, conf=None, **kwargs):
          returns H2O Context
         """
 
+        # When external cluster handle is specified then we need to make sure the configuration
+        # is corresponding to this situation -> We already have H2O cluster running and need to connect to it.
+        # For this we need to use manual cluster start mode, need to know the cloud name and ip address and port of a
+        # single node in this cluster
+        if external_cluster_handle is not None:
```

Comment:
Would be great to test this path in cooperation with @CreatureDev . A possible problem i can see, is that we are forwarding comm via steam proxy. Probably not big deal, however, in theory proxy port does not need to be visible on edge node side - we need to just test it.

Reply:
If I understand correctly, only communication with REST API should be transmitted using steam proxy. The rest of internal communication is done via internal protocol ( h2o client <-> h2o worker nodes) and should not be affected by this.

Just to understand this properly, why the proxy port might not be visible on the edge node ?